src/paramsUi.cpp

```diff
@@ -549,8 +551,12 @@ class ParamsDialog {
 			}
 			return 1; // Eat the keystroke.
 		}
-		if (msg->wParam == VK_SPACE) {
-			// Let REAPER handle the space key so control+space works.
+		if (msg->wParam == VK_SPACE && (
+			msg->hwnd == dialog->moreButton ||
```

Comment:
I don't understand why we we need to restrict this when we didn't before. Previously, we always passed the space key through to REAPER. Now, we're passing it through in less situations than we did before, not more. Can you explain?

Reply:
> Previously, we always passed the space key through to REAPER.
> 
That relied on what Cockos described as an "ancient hardcoded ctrl+space playback binding".
> Now, we're passing it through in less situations than we did before, not more. Can you explain?
> 
The hardcoded binding was removed in REAPER 7.40, so OSARA needs to handle transport differently in our params dialogs now.

Reply:
That doesn't really answer my question, though. We're actually more tightly restricting this key with this change, not less tightly restricting it. As I understand it, this change shouldn't be necessary at all; space should always have worked. I'm not arguing that this code doesn't work - I'm sure you've tested it - but I also don't think it's correct, and that makes me nervous about weird regressions down the line.